src/components/App.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
-// import logo from '../logo.svg';
-import "../App.css";
 import Display from "./Display";
+import "../App.css";
 import ButtonPanel from "./ButtonPanel";
+// eslint-disable-next-line no-unused-vars
+import calculate from "../logic/calculate";
 
```

Comment:
- [ ] this can be understood because you are not yet using calculate.js either do not forget to remove this once you begin using calculate.js